Userland/Utilities/js.cpp

```diff
@@ -614,32 +615,37 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     args_parser.add_option(gc_on_every_allocation, "GC on every allocation", "gc-on-every-allocation", 'g');
     args_parser.add_option(disable_syntax_highlight, "Disable live syntax highlighting", "no-syntax-highlight", 's');
     args_parser.add_option(evaluate_script, "Evaluate argument as a script", "evaluate", 'c', "script");
+    args_parser.add_option(disable_debug_printing, "Disable debug output", "disable-debug-output", {});
```

Comment:
Please keep options in the same order as their value declarations above (before `evaluate_script`)

Reply:
Fixed, thanks

Reply:
Fixed, thanks!